pkg/cmd/completion.go

```diff
@@ -33,6 +34,53 @@ func newCompletionCmd() *completionCmd {
 	return cc
 }
 
+var zsh_completion_instructions = `
+Suggested next steps:
+----------------------
+1. Move ` + "`stripe-completion.zsh`" + `to the correct location:
+    mkdir -p ~/.stripe
+    mv stripe-completion.zsh ~/.stripe
+
+2. Add the following lines to your ` + "`.zshrc`" + ` enabling shell completion for Stripe:
+    fpath=(~/.stripe $fpath)
+    autoload -Uz compinit && compinit -i
+
+3. Source your ` + "`.zshrc`" + `or open a new terminal session:
+    source ~/.zshrc
+`
+
+var bash_completion_instructions_mac = `
+1. (If you've never installed bash completion before) Install bash completion on your system.
+    brew install bash-completion
+
+2. (If you've never installed bash completion before) Follow the post-instructions that brew displays, and add a line like the following to your bash profile:
```

Comment:
In `(If you've never installed bash completion before)` should we change `bash completion` to `bash-completion`? Would need to be changed in step 1 as well.